Sources/ReactiveMoya/ReactiveSwiftMoyaProvider.swift

```diff
@@ -17,6 +18,11 @@ open class ReactiveSwiftMoyaProvider<Target>: MoyaProvider<Target> where Target:
         self.stubScheduler = stubScheduler
         super.init(endpointClosure: endpointClosure, requestClosure: requestClosure, stubClosure: stubClosure, manager: manager, plugins: plugins, trackInflights: trackInflights)
     }
+    
+    fileprivate init(moyaProvider: MoyaProvider<Target>) {
+        self.stubScheduler = nil
```

Comment:
Hmm, because of the queues we're adding in the other PR, I think this scheduler could be removed and then making this extensions would be quite easy.